packages/@adobe/spectrum-css-temp/components/circleloader/skin.css

```diff
@@ -31,6 +32,16 @@ governing permissions and limitations under the License.
   }
 }
 
+/* TODO: use semantic tokens or spectrum variables when available */
+.spectrum-CircleLoader--staticBlack {
+  .spectrum-CircleLoader-track {
+    border-color: rgba(0, 0, 0, 0.25);
```

Comment:
you could copy what we do here https://github.com/adobe/react-spectrum/blob/main/packages/%40adobe/spectrum-css-temp/components/button/skin.css#L298

i'd make this a variable and use that instead of directly setting the property